src/main/java/mekanism/common/lib/multiblock/Structure.java

```diff
@@ -149,8 +149,8 @@ public void add(Structure s) {
                         //Otherwise, if the minor map does have an entry, merge them
                         minorPlane.merge(value);
                         //If after merging the planes
-                        if (minorPlane.hasFrame() && minorPlane.length() >= 2 && minorPlane.height() >= 2) {
-                            // the plane has a frame and is at least two by two
+                        if (minorPlane.hasFrame()) {
+                            // the plane has a frame
```

Comment:
I think this change has some significant implications and in fact may entirely invalidate the point/efficiency of having two plane types (though @aidancbrady can probably say better)